File: python/notification.py
class Notification:
    def __init__(self, notification_id, user_id, message, date, status):
        self.notification_id = notification_id
        self.user_id = user_id
        self.message = message
        self.date = date
        self.status = status

    def send_notification(self, user_id, message):
        print(f"Notification sent to user ID: {user_id}")

    def mark_as_read(self, notification_id):
        print(f"Notification ID {notification_id} marked as read.")

    def get_notifications(self, user_id):
        print(f"Getting notifications for user ID: {user_id}")

File: GoaOnline/src/Notification.java
public class Notification {
    private int notificationId;
    private int userId;
    private String message;
    private String date;
    private String status;

    // Constructor
    public Notification(int notificationId, int userId, String message, String date, String status) {
        this.notificationId = notificationId;
        this.userId = userId;
        this.message = message;
        this.date = date;
        this.status = status;
    }

    // Methods
    public void sendNotification(int userId, String message) {
        System.out.println("Notification sent to user ID: " + userId);
    }

    public void markAsRead(int notificationId) {
        System.out.println("Notification ID " + notificationId + " marked as read.");
    }

    public void getNotifications(int userId) {
        System.out.println("Getting notifications for user ID: " + userId);
    }
}
